CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.5.1
+
+* CLI
+  * Added a `linkerd check config` command for verifying that
+    `linkerd install config` was successful
+  * Improved the help documentation of `linkerd install` to clarify flag usage
+  * Added support for private Kubernetes clusters by changing the CLI to connect
+    to the control plane using a port-forward (thanks, @jackprice!)
+* Controller
+  * Fixed pod creation failure when a `ResourceQuota` exists by adding a default
+    resource spec for the proxy-init init container
+* Proxy
+  * Replaced fixed reconnect backoff with exponential one (thanks,
+    @zaharidichev!)
+  * Support endpoint weights
```

Comment:
How about "Added support for endpoint weights" for consistency with the tense of the other bullet points?

Reply:
Whoops, meant to remove this since it's not a usable end-to-end feature yet.